apps/aecore/src/aec_peer_connection.erl

```diff
@@ -1443,3 +1461,34 @@ validate_micro_signature(MicroHeader, _PrevHeader, PrevKeyHeader)
     end;
 validate_micro_signature(_MicroHeader, _PrevHeader, not_found) ->
     {error, signer_not_found}.
+
+%% -- Node info  message -----------------------------------------------------------
+handle_get_node_info(S) ->
+    case is_node_info_sharing_enabled() of
+        false -> S;
+        true ->
+            NodeVersion = aeu_info:get_version(),
+            NodeRevision = aeu_info:get_revision(),
+            OS = aeu_info:get_os(),
+            Verified = aec_peers:count(verified),
+            Unverified = aec_peers:count(unverified),
+            NodeInfo = #{ version => NodeVersion
```

Comment:
Revision might be an overkill, the vendor is lacking - what if alternate AE implementations pop up?
https://github.com/ethereum/devp2p/blob/master/rlpx.md#hello-0x00

Reply:
added a vendor